flake.nix

```diff
@@ -1,19 +1,78 @@
 {
-  description = "Gomuks development environment";
+  description = "Gomuks development environment & packages";
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
     flake-utils.url = "github:numtide/flake-utils";
   };
 
-  outputs = { self, nixpkgs, flake-utils }:
-    (flake-utils.lib.eachDefaultSystem (system:
+  outputs =
+    {
+      self,
+      nixpkgs,
+      flake-utils,
+      ...
+    }:
+    flake-utils.lib.eachDefaultSystem (
+      system:
       let
         pkgs = import nixpkgs {
           inherit system;
-          config.permittedInsecurePackages = [ "olm-3.2.16" ];
+          config = {
+            permittedInsecurePackages = [ "olm-3.2.16" ];
+          };
+        };
+
+        outPackages = self.outputs.packages.${system};
+
+        # Extract version from version.go
+        versionContent = builtins.readFile ./version/version.go;
+        versionMatch = builtins.match ''^.*const StaticVersion = "([0-9\.]+)".*$'' versionContent;
+        version = builtins.elemAt versionMatch 0;
+      in
+      {
+        packages = {
+          gomuks = pkgs.buildGoModule {
+            pname = "gomuks";
+            inherit version;
+
+            src = ./.;
+
+            # Go dependency hash (should be updated when dependencies are)
```

Comment:
The reason I didn't add a package to the flake was because of this. I don't want to have to make a PR every time we update dependencies.

Reply:
Tulir doesn't use nix. I know from experience that it is very annoying to manage the vendor hash even when using nix. I forget to update the hash consistently on my own projects.